frontend/src/components/News/News.js

```diff
@@ -0,0 +1,133 @@
+import {React,useState} from 'react'
+import { Table,Navbar,Container } from 'react-bootstrap';
+import { BsSearch } from "react-icons/bs";
+import axios from 'axios';
+import logo from '../../../src/images/Stocks_GrowthCombined.png';
+import "./News.css"
+
+
+export default function News() {
+
+    const [searchWord, setSearchWord] = useState("");
+    const [searchResult, setSearchResult] = useState({})
+
+    const [searchResultGlobal , setSearchResultGlobal] = useState({})
+
+
+    const searchRequest = {
+        method: 'GET',
+        url: 'https://newsapi.org/v2/everything',
+        params: {q: searchWord, sortBy: 'publishedAt', apiKey: "360ced0df45347fdb65522b2e8bf373a" },
+      };
+
+    
+    const globalNews = {
+        method: 'GET',
+        url: 'https://newsapi.org/v2/everything',
+        params: {q: "global news", sortBy: 'publishedAt', apiKey: "360ced0df45347fdb65522b2e8bf373a" },
+      };
+
+
+
+    const searchAction = (e) => {
+
+        if (e.type === "click" || e.key === 'Enter') {
+            axios.request(searchRequest).then(function (response) {
+                setSearchResult(response.data);
+            }).catch(function (error) { 
+                console.error(error);
+
+            })
+        }
+        axios.request(globalNews).then(function (response) {
```

Comment:
This can be better implemented with the useEffect from React, which will load the global news before the page renders, instead of loading when the user presses a key for the first time. Also, in the way it is implemented now, every time the user presses a key it fetches the same information unnecessarily. The useEffect function can prevent that continuos re-fetching of the news.

Reply:
Thanks for the information.
Fixed.